rx-examples/src/main/java/io/vertx/example/rxjava/database/jdbc/Streaming.java

```diff
@@ -24,26 +24,16 @@ public void start() throws Exception {
     JDBCClient jdbc = JDBCClient.createShared(vertx, config);
 
     // Connect to the database
-    jdbc.rxGetConnection().subscribe(
-      conn -> {
-
-        conn.rxUpdate("CREATE TABLE test(col VARCHAR(20))").
-          flatMap(result -> conn.rxUpdate("INSERT INTO test (col) VALUES ('val1')")).
-          flatMap(result -> conn.rxUpdate("INSERT INTO test (col) VALUES ('val2')")).
-          flatMap(result -> conn.rxQueryStream("SELECT * FROM test")).subscribe(
-            rowStream -> {
-              rowStream.toObservable().subscribe(
-                row -> System.out.println("Row : " + row.encode()),
-                Throwable::printStackTrace,
-                conn::close
-              );
-            },
-          Throwable::printStackTrace);
-      },
-
-      // Could not connect
-      Throwable::printStackTrace
-    );
-
+    jdbc.rxGetConnection().
+        flatMap(conn -> conn.rxUpdate("CREATE TABLE test(col VARCHAR(20))").
+        flatMap(result -> conn.rxUpdate("INSERT INTO test (col) VALUES ('val1')")).
+        flatMap(result -> conn.rxUpdate("INSERT INTO test (col) VALUES ('val2')")).
+        flatMap(result -> conn.rxQueryStream("SELECT * FROM test"))).subscribe(
```

Comment:
You should not subscribe inside subscribe, otherwise you break the flow.

`Single` has a `flatMapObservable` method you can combine with `rowStream.toObservable`

See https://github.com/tsegismont/vertx-musicstore/blob/master/src/main/java/io/vertx/demo/musicstore/AlbumHandler.java#L83-L84